packages/dds/tree/src/feature-libraries/sequence-field/sequenceFieldEditor.ts

```diff
@@ -47,6 +64,80 @@ export const sequenceFieldEditor = {
                   detachedBy: revision,
                   detachIndex,
               }),
+    move(sourceIndex: number, count: number, destIndex: number): Changeset<never> {
+        if (count === 0 || sourceIndex === destIndex) {
+            // TODO: Should we allow creating a move which has no observable effect?
```

Comment:
It seems to make the API easier to work with if we do.
Also, for the `sourceIndex === destIndex`, it can have effects in the presence of concurrent changes (it should bring back those nodes that had been concurrently moved away), which makes me think `return []` is not what we want for that case.